### ISSUE TITLE
Post linter

### ISSUE THREAD

**Issue by abraham** [forem/forem#1449]
**Is your feature request related to a problem? Please describe.**

Sometimes posts have simple mistakes like no syntax highlighting.

**Describe the solution you'd like**

It would be nice if there was some markdown linting happening in the post editor that would suggest improvements to your post. A few issues that it might be useful to highlight with a linter:

- Code blocks without language tags
- Duplicate title of an existing post
- Missing tags
- Images tags with relative paths
- Images without alt text
- The same image alt text used multiple times
- Things that could be liquid tags like GitHub/Twitter links

**Comment by Link2Twenty** [comment]
A phase 2 of this could be autocompleting certain inputs, usernames, liquid tags etc

Kinda like github do

![image](https://user-images.githubusercontent.com/3534427/50765601-513a7980-126e-11e9-9ee3-9eb368fed92f.png)

**Comment by kreopelle** [comment]
@benhalpern  - I would be interested in taking this on, perhaps breaking it into smaller chunks.

**Comment by cmgorton** [comment]
Hello! Thanks for your contributions here.  As a heads up, I am going to close this issue.  We have a new process moving forward to address feature requests and especially those that have been in our repo for 1+ years without much traction.  

**TLDR:** we have new internal RFC process for feature requests. Any features that are more substantial and require more thought and detail will need to be discussed in [forem.dev](forem.dev).

You can read more about our Internal RFC Process and forem.dev Discussions in [this post](https://forem.dev/foremteam/internal-rfc-process-and-forem-dev-discussions-3gl4).
